h/views.py

```diff
@@ -1,16 +1,18 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import logging
-import re
 
 from pyramid import httpexceptions
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client
```

Comment:
Hmm, actually `import .api_client` doesn' seem to work, I guess relative imports only work with `from`. I think they're a bad idea but I'll do it if you guys really insist.